api/src/main/java/org/asynchttpclient/request/body/generator/IFeedableBodyGenerator.java

```diff
@@ -0,0 +1,11 @@
+package org.asynchttpclient.request.body.generator;
```

Comment:
Missing license header. Please grant copyright to "the AsyncHttpClient Project".